oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/AnalyzerModuleProvider.java

```diff
@@ -111,18 +107,17 @@ public void prepare() throws ServiceNotProvidedException, ModuleStartException {
     public void start() throws ModuleStartException {
         // load official analysis
         getManager().find(CoreModule.NAME)
-                    .provider()
-                    .getService(OALEngineLoaderService.class)
-                    .load(CoreOALDefine.INSTANCE);
+                .provider()
+                .getService(OALEngineLoaderService.class)
+                .load(CoreOALDefine.INSTANCE);
```

Comment:
Please import our code style settings, and be aware of these unrelated changes when submitting an PR

Reply:
Where can I get the `code style settings` ?

Reply:
https://github.com/apache/skywalking/blob/master/codeStyle.xml